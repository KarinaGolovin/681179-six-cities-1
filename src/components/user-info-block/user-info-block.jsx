import React from 'react';
import PropTypes from 'prop-types';

export const UserInfoBlock = ({avatar, email}) => {
  return (
    <>
      <div className="header__avatar-wrapper user__avatar-wrapper">
        <img src={avatar}/>
      </div>
      <span className="header__user-name user__name">{email}</span>
    </>
  );
};

UserInfoBlock.propTypes = {
  email: PropTypes.string,
  avatar: PropTypes.string
};
