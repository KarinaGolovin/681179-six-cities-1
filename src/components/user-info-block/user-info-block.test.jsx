import React from 'react';
import renderer from 'react-test-renderer';
import {UserInfoBlock} from './user-info-block.jsx';

const mock = {
  avatar: ``,
  email: ``,
};

it(`UserInfoBlock correctly renders`, () => {
  const component = renderer.create(
      <UserInfoBlock
        avatar={mock.avatar}
        email={mock.email}
      />
  ).toJSON();

  expect(component).toMatchSnapshot();
});
