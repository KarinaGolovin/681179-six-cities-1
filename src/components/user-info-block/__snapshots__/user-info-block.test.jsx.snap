// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfoBlock correctly renders 1`] = `
Array [
  <div
    className="header__avatar-wrapper user__avatar-wrapper"
  >
    <img
      className="user__avatar"
      src=""
    />
  </div>,
  <span
    className="header__user-name user__name"
  >
    
  </span>,
]
`;
